csfieldguide/static/interactives/number-memory/scss/number-memory.scss

```diff
@@ -0,0 +1,30 @@
+.number-memory {
```

Comment:
The border around this makes it look super inconsistent with the rest of the in-page interactives. I'm not a huge fan of it in whole-page mode either but that could just be personal preference.

When I first saw it I immediately thought it was a link to a whole page interactive because when you hover over one of those links it has a similar appearance.

Reply:
I don't mind it around the distraction text though. I think the distraction text _does_ need something to kind of separate it from the chapter text (like the border).